rasa/nlu/training_data/formats/markdown.py

```diff
@@ -37,11 +35,8 @@
 REGEX = "regex"
 LOOKUP = "lookup"
 available_sections = [INTENT, SYNONYM, REGEX, LOOKUP]
+markdown_section_markers = [f"## {s}:" for s in available_sections]
```

Comment:
this is used as a constant and imported in another module (in `loading.py`), so should be capitalised IMO

Reply:
good point!